aiohttp/helpers.py

```diff
@@ -784,11 +786,41 @@ def set_exception(fut: "asyncio.Future[_T]", exc: BaseException) -> None:
         fut.set_exception(exc)
 
 
+@functools.total_ordering
+class AppKey(Generic[_T]):
```

Comment:
I initially put this in web_app.py, but that looks like a circular import to reference it here. Maybe I can still move it back though and do a `if TYPE_CHECKING` thing here.

Reply:
I think this is fine